spec/features/albums_can_be_created_and_viewed_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+feature 'Album' do
+  let!(:user) { create(:user) }
+  let(:test_album_name) { 'My days at Makers' }
+
+  before do
+    visit("/users/sign_in")
+    fill_in 'user_email', with: user.email
+    fill_in 'user_password', with: user.password
+    within(:css, 'div.actions') do
+      click_button 'Log in'
+    end
+  end
+
+  scenario 'View albums' do
```

Comment:
`scenario 'View albums'`

this is testing that there are two links on the page, which is good but a different test.

for this test, i'd suggest to use the factory and create an album and then assert that its visibile to the user